Skip to Main ContentSkip to Main Menu
Advertisement

Trends in Molecular Medicine
Submit

Simon Wang
This journal
Journals
Publish
News & events
About Cell Press
Search for...

Advanced search
ReviewVolume 27, Issue 11p1060-1073November 2021Open access
Download Full Issue
Enhancers in disease: molecular basis and emerging treatment strategies
Annique Claringbould ∙ Judith B. Zaugg judith.zaugg@embl.de

Affiliations & Notes

Article Info
Cover Image - Trends in Molecular Medicine, Volume 27, Issue 11
Download PDF
Cite

Share
Set Alert
Get Rights
Reprints
Previous article
Next article

Show Outline
Highlights
Enhancer disruption is increasingly implicated as a disease-driving mechanism. Chromosomal rearrangements can cause an enhancer to drive aberrant gene expression, genetic variants in enhancers can impact a transcription factor binding site, and disease-associated epigenetic changes are enriched in enhancer regions.
The three big challenges in enhancer research focus on systematically identifying functional enhancers, their target genes, and the context in which they are active.
Bromo- and extra-terminal (BET) inhibitors are a new class of drugs that target enhancers and inhibit gene expression. They are under investigation as treatment for cancer and other diseases.
Gene editing techniques elucidate enhancer function and are being used to selectively regulate or mutate disturbed enhancers.
Abstract
Enhancers are genomic sequences that play a key role in regulating tissue-specific gene expression levels. An increasing number of diseases are linked to impaired enhancer function through chromosomal rearrangement, genetic variation within enhancers, or epigenetic modulation. Here, we review how these enhancer disruptions have recently been implicated in congenital disorders, cancers, and common complex diseases and address the implications for diagnosis and treatment. Although further fundamental research into enhancer function, target genes, and context is required, enhancer-targeting drugs and gene editing approaches show great therapeutic promise for a range of diseases.
Keywords
enhancer
gene regulation
complex disease
enhanceropathy
BET inhibition
genome-wide association studies
chromatin organization
structural variation
epigenetic drugs
Enhancers and their role in human disease
Gene expression is regulated by numerous factors, including polymerase recruitment, epigenetic signaling, and transcription factors (TFs) (see Glossary) that regulate the activity of gene promoters and enhancers. Enhancers are regulatory sequences in the genome that affect gene expression of a nearby gene. The exact definition of enhancers and the mechanism by which they regulate gene expression is still a matter of active research (Box 1). The current view is that they act by recruiting specific TFs and polymerase to a distal site, which then activate gene expression through a mechanism that involves physical contact with the gene promoter (Figure 1A). More recently, it has been proposed that enhancers can form highly active genomic clusters (super enhancers or stretch enhancers [1]), which may act in a phase-separated assembly of molecules [2].
Figure 1
Figure 1 Cartoon representation of an enhancer loop and how it can be disrupted.
Show full captionFigure viewer
Box 1
Enhancer features, function, and identification
Enhancers can regulate gene expression by recruiting TFs and the transcriptional machinery and subsequently forming a loop with the promoter region of the target gene (see Figure 1A in main text) [87]. Chromatin is spatially organized into large domains called TADs, which comprise smaller loops, including enhancer–promoter loops [88–91]. The cohesin complex demarcates TAD boundaries and forms smaller enhancer–promoter loops by active extrusion of chromatin through its ring-shaped structure [92,93]. Chromatin looping allows enhancers to target distal genes, although most target genes are located nearby on the linear chromosome [94]. The effect of chromatin organization on gene expression is still under debate, as reviewed in [89,95]. On the one hand, cohesin-mediated enhancer–promoter loops across cell lines correlate with gene expression [96] and mutations to the cohesin complex result in loss of gene expression [97]. On the other hand, multiple studies have shown that changes in TAD organization do not translate to changes in gene expression [98,99].
An enhancer sequence can be validated experimentally by measuring its ability to drive expression of a reporter gene on a plasmid [100] or inserted into the genome [101]. However, it is challenging to investigate enhancer activity in their endogenous locus. Recent advances in CRISPR methods have started to address this by inducing or repressing specific enhancer elements in endogenous loci [41,42,102] and measuring gene expression. A complementary approach is to predict endogenous enhancers based on their biochemical properties. Enhancers are characterized by specific chromatin modifications like H3K4me1 and H3K27ac [103–105]. Active enhancers are often flanked by bidirectional capped RNAs, indicating that they can activate transcription both upstream and downstream [84,106].
Enhancers can form highly active genomic clusters (super enhancers or stretch enhancers [1]) with a shared function in cell type-specific gene regulation, although it is not yet clear if the individual enhancers each play a separate role or whether they are mostly redundant [107,108]. Super enhancers seem particularly vulnerable to the loss of chromatin structure, for example, by downregulating cohesin and disturbing chromatin organization [92]. Enhancer clusters may act in a phase-separated assembly of molecules [2]: modifications of TFs, enhancer sequences, and other regulatory factors form temporary links that result in a phase-separated multi-molecular complex of transcriptional regulators that drives cooperativity within a super enhancer [2,109].
Shortly after the first enhancer sequence was described to affect the expression of a rabbit hemoglobin β-1 gene [3], evidence that enhancer disruption could lead to disease in humans was found, with the misregulation of the human β-globin gene by a DNA translocation in patients suffering from β-thalassemia [4]. Since then, a number of other enhanceropathies have been identified [5], including polydactyly caused by mutations 1 Mb away from the Sonic hedgehog gene [6] and the translocation of the IgH enhancer resulting in overexpression of MYC in Burkitt’s lymphoma [7].
With the advent of genome-wide association studies (GWAS) that identify associations between genetic variants and complex traits and diseases, it became clear that the majority of trait-linked genetic variants lie in noncoding genomic regions far from promoters, thus likely targeting enhancers. These associations, together with an increasing number of enhancer-mediated mechanisms driving cancer and rare diseases, suggest that enhancers are on the way to becoming the next big frontier in drug target identification.
Structural variation leading to enhancer mistargeting
Enhancer hijacking is an important cancer-driver mechanism
Structural variation may induce ‘enhancer hijacking’ if an active enhancer is misplaced such that it regulates genes that are not their original targets [8]. This phenomenon has been described in cancer, where enhancers can be misplaced through chromosomal rearrangements [8] or micro-amplification of genomic regions [9], to induce the expression of an oncogene (Table 1). A recent analysis of whole-genome sequencing profiles in over 1200 cancer genomes found hundreds of genes for which expression alterations were observed within 100 kb of a structural variation breakpoint, some of which could be attributed to enhancer hijacking [10]. These data indicate that enhancer hijacking may be a more common mechanism in cancer than we currently appreciate based on the few validated examples. Yet the dual challenge of identifying functional enhancers and determining the cancer-driving potential of mutations makes this phenomenon particularly hard to study.
Disruption mechanism	Phenotype	Description	Refs
Enhancer hijacking	Medulloblastoma	Somatic genomic rearrangements result in a fusion of an active enhancer with oncogenes GFI1 or GFI1B	[8]
Enhancer hijacking	(Pediatric) cancers	Amplification of MYCN can be regulated by local and distal enhancers	[113]
Enhancer hijacking	Salivary gland acinic cell carcinoma	Rearrangements translocate active enhancer to activate NR4A3, a TF that then upregulates its target genes	[114]
Enhancer hijacking	Multiple cancers	18 candidate enhancer hijacking events in a pan-cancer analysis and 98 in tumor-type specific analyses (including IGF2 in colorectal cancer)	[115]
Enhancer hijacking	Primary gastric adenocarcinoma	Rearrangements lead to enhancers mistargeting CCNE1 and IGF2	[116]
Enhancer hijacking	T-lineage acute lymphoblastic leukaemia	Translocations lead to enhancer hijacking of the HOXA gene cluster	[117]
Enhancer hijacking	T-lineage acute lymphoblastic leukemia	Small insertions in the enhancers that regulate oncogenes (e.g. LMO2) drive aberrant expression	[9]
TAD boundary removal	Limb malformations	TAD boundary removed by rearrangements, subsequent fusion of two TADs, enhancer Epha4 regulates genes in both domains leading to ectopic expression	[12]
TAD boundary removal	5q14.3 microdeletion syndrome	Disrupted TAD structure in patients leads to decreased expression of MEF2C, a known disease gene	[118]
TAD boundary removal	Fragile X syndrome	CTCF binding impaired by CGG triplet repeat near FMR1	[13]
Mutation in enhancer	Pierre Robin syndrome	Microdeletion in a developmental enhancer that regulates SOX9 in the nervous system and skeletal structures	[119]
Mutation in enhancer	Isolated atrial defects	Homozygous point mutation located in a highly conserved enhancer region 90 kb downstream of the developmental transcription factor TBX5	[120]
Mutation in enhancer	Isolated pancreatic agenesis	Mutations within PTF1A enhancer	[121]
Mutation in enhancer	Intellectual disability	Enrichment of de novo mutations in fetal brain-specific enhancers, identification of enhancer that regulates CSMD1 and affects neurogenesis	[21]
Enrichment of SNPs in enhancers	Crohn’s disease, multiple sclerosis, electrocardiogram phenotypes	Cell type-specific enrichment of GWAS SNPs in DNase I hypersensitivity sites of Th17, CD3+, and heart cell, respectively	[122]
Enrichment of SNPs in enhancers	Immune diseases	60% of GWAS fine-mapped variants located in immune cell enhancers	[123]
Enrichment of SNPs in enhancers	Type I diabetes	GWAS SNPs enriched in thymus, CD4+ and CD8+ T cells, B cells, and CD34+ enhancers	[124]
Enrichment of SNPs in enhancers	Type II diabetes	GWAS SNPs enriched in pancreatic islet enhancer regions	[125]
Enrichment of SNPs in enhancers	Immunodeficiency	Two enhancers influencing RASGRP1 levels enriched for autoimmunity-associated SNPs	[126]
Enrichment of SNPs in enhancers	Hirschsprung disease	SNPs located in RET enhancers are necessary but not sufficient to develop the disease and interact synergistically	[39]
Epigenetic enhancer misregulation	Pulmonary arterial hypertension	Patients' pulmonary arterial endothelial cells show widespread remodeling of active chromatin regions (H3K27ac), specifically of enhancers	[47]
Differentially methylated enhancers	Osteoarthritis	Patients' cartilage chondrocytes contain 17 000 differentially methylated CpGs, almost half are located in enhancer loci	[50]
Differentially methylated enhancers	Alzheimer's disease	Patients' neurons have differentially methylated enhancers	[127]
Differentially methylated enhancers	Wilson disease	Patients' liver samples contain hypermethylated regions that are enriched for (liver) enhancers	[128]
Differentially methylated enhancers	Multiple sclerosis	Monozygotic twins discordant for disease, handful of differentially methylated CpG sites, one located in an enhancer that controls ZBTB16 was linked to glucocorticoid treatment of the co-twin with multiple sclerosis	[129]
Differentially methylated enhancers	Ewing sarcoma	Disease-specific hypomethylated CpGs were enriched in disease-specific enhancers	[130]
Table 1
Examples of enhancer disruptions driving diseasea
a
Abbreviations: CCNE1, cyclin E1; CpG, cytosine and guanine nucleotide sequence; CSMD1, CUB and Sushi Multiple Domains 1; CTCF, CCCTC-binding factor; FMR1, Fragile X Mental Retardation Protein 1; GFI1, Growth Factor Independent 1; GWAS, genome-wide association study; HOXA, Homeobox A; IGF2, insulin-like Growth Factor 2; LMO2, LIM Domain Only 2; MEF2C, Myocyte Enhancer Factor 2C; PTF1A, Pancreas Associated Transcription Factor 1A; RASGRP1, RAS Guanyl Releasing Protein 1; RET, Ret Proto-Oncogene; SNP, single nucleotide polymorphism; SOX9, SRY-Box Transcription Factor 9; TAD, topologically associated domain; TBX5, T-Box Transcription Factor 5; TF, transcription factor; ZBTB16, Zinc Finger And BTB Domain Containing 16.
Open table in a new tab
Few diseases are caused by disruption of TAD boundaries
Another way that enhancers may mistarget their genes is through disruption of boundaries of chromatin domains, termed topologically associated domains (TADs) (Box 1). Groups of coregulated genes, often targeted by common enhancers, cluster within TADs [11]. Consequently, disruption of a TAD boundary can lead an enhancer to interact with genes outside of the original TAD, which was identified as a disease mechanism in three congenital disorders, including limb malformation [12] (Table 1). Moreover, disease-associated short tandem repeats appear to colocalize with CpG-rich TAD boundaries, suggesting that 3D chromatin structure may affect disease status [13]. However, while a large number of disease-related copy number variations (CNVs) are predicted to affect TAD structure [14,15], a whole-genome analysis in over 2500 cancer genomes identified almost 300 000 TAD disruptions, of which only 14% led to a change in gene expression [16]. Similarly, a recent study found that 41% of TADs across 36 datasets had at least one boundary disrupted by a common CNV classified as (likely) benign [17], indicating that TAD boundary deletion without a strong effect on the phenotype is widespread. Given the small numbers of diseases associated with a TAD boundary disruption, it appears that either such disruptions are too rare to be identified, too disruptive to produce a viable embryo, or not important enough to cause a detectable phenotype. While the final conclusion may very well be a mix of these, for now, TAD boundary disruptions seem to be a very rare disease mechanism.
Mutations in enhancers leading to congenital disorders
The total number of conserved variants within a gene’s enhancers are a good predictor of its pathogenicity [18], suggesting that mutations in bona fide enhancers likely contribute to the development of congenital disease. A number of enhancer mutations lead to congenital disorders (Table 1; more examples in [19]), including several neurological conditions, as recently reviewed in [20]. One challenge in investigating enhanceropathies is that the effect of the responsible enhancer may be cell- and/or developmental-stage specific and may thus not be measurable in the differentiated tissue. The limited number of known enhancer-driven congenital disorders may therefore reflect the lack of annotated developmental enhancers. This is illustrated by a recent study that found de novo mutations in individuals with intellectual disability specifically enriched in fetal brain enhancers that affected neurogenesis [21]. Overall, the fraction of congenital disorders with an underlying link to enhancer disruption is likely much higher than currently known and integration of large-scale development- and tissue-specific enhancer atlases [22,23] with whole-genome sequencing of patients will be crucial to identify them.
Enhancer misregulation in common complex diseases
GWAS have identified hundreds of thousands of genetic risk factors for a host of diseases. While a minority of those disease-associated variants alter the protein-coding sequence (Figure 2), the majority (91%) fall in noncoding regions far from promoters, suggesting an enhancer-mediated regulatory role.
Figure 2
Figure 2 Genome-wide association study (GWAS) SNPs almost always fall in regulatory regions.
Show full captionFigure viewer
Genetic misregulation of enhancers
Many studies have observed enrichment of GWAS SNPs within putative enhancers in cell types that match with the expected disease biology (some examples in Table 1) [24]. Yet, it remains an open challenge to translate GWAS SNPs in enhancer regions into disease mechanisms, mostly due to the challenges in basic enhancer biology: it is unclear which genes they target and in what tissue. For diseases where the cell type of origin is unclear or unknown, overlapping GWAS SNPs with cell type-specific enhancers provides a strategy for identifying the disease-relevant cell type(s) [24], which is crucial for developing potential therapeutic strategies. One example where GWAS results have revealed an unexpected disease origin is in Alzheimer’s disease. Initially, neurons were considered the cells of interest, yet most disease-associated variants were found near genes that are active in immune cells [25], shifting the research focus away from neurons towards the immune system [26].
There are many strategies to identify putative disease genes from GWAS results. Colocalizing or overlapping GWAS SNPs with expression quantitative trait loci (eQTL) is a commonly used approach, because it allows prioritization among several genes in a GWAS locus [27,28]. However, eQTLs can be tissue-specific and the effect of the genetic variant may have happened already in earlier developmental stages. In this light, it may not be surprising that prioritization of known disease genes by local eQTL colocalization does not outperform taking the gene with the closest transcription start site to the GWAS variant [18,29]. This is consistent with the estimation that only 11% of disease heritability is mediated through the gene expression of local genes [30] and that local eQTLs with large effect sizes on disease-causing genes are likely removed by negative selection [31]. Colocalizing GWAS results with trans-eQTLs, where a SNP correlates with distal gene expression, may improve identification of disease genes and the distal enhancers driving them [32]. Indeed, the intersection of tissue-specific eQTL results, fine-mapped GWAS summary statistics, and enhancer sequences may be a valuable strategy to zoom into disease mechanisms [29,33]. A second strategy to link GWAS SNPs in enhancers to their putative target genes is to use information from 3D chromatin capture technologies, such as Hi-C, ChIA-PET, or Capture-C to map the chromatin interactions in the disease-relevant tissue [34,35]. This strategy has been applied in recent GWAS studies [36,37] and has led to identification of genes that were not in linkage disequilibrium with the SNP, yet very plausible based on their biological function [37]. While each method has technical limitations that complicate target gene identification, this field is developing rapidly, as reviewed in [38].
An additional challenge in interpreting GWAS signals is that each genetic variant on its own typically has a very low risk, which highlights the importance of investigating multiple genetic variants in enhancers in parallel. An illustrative example for this is Hirschsprung disease, a multifactorial bowel disease that occurs as syndromic and isolated forms. A number of known rare coding variants and common regulatory risk factors contribute to disease risk [39]. The common SNPs, located in RET enhancers, are necessary but not sufficient to develop the disease and interact synergistically with the rare coding variants. Another way to investigate the role of multiple variants simultaneously is through polygenic risk scores (PRSs), as they reflect the genetic risk from all disease-associated loci. A recent study of type II diabetes (T2D), leveraged PRSs to conclude that a large part of disease heritability in T2D is mediated through islet hub enhancer loci [36], again highlighting the importance of enhancers in disease mechanisms. The idea that many loci with small individual effects should be investigated simultaneously is in line with the recently proposed omnigenic model, which postulates that common complex disease is mediated through the gene expression of a select group of core genes that are in turn connected to all peripheral genes expressed in the disease-relevant tissue(s) [40]. The omnigenic model thus explains how thousands of GWAS hits with very small effect sizes can have a large impact on the phenotype.
Despite the enrichment of enhancers in GWAS loci, their role in these highly polygenic diseases is not entirely clear yet. Partially, this is because individual GWAS loci typically contribute very little to disease heritability and their mechanistic dissection requires very demanding in-depth experimental strategies that may be difficult to justify given their small effect size. Still, it is likely that the combinatorial effects of many enhancers play an important role in disease biology and, if the omnigenic model holds, it is essential to uncover how enhancers drive the expression of core disease genes. We expect that high-throughput validation approaches, for example, building on recent technologies of CRISPRi screens in enhancers [41,42], to test the functional impact of a large number of SNPs will soon become feasible. This will provide more direct estimates of the extent to which enhancers affect disease biology.
Epigenetic misregulation of enhancers
Enhancer function can also be affected by nongenetic factors such as chromatin remodeling or DNA modifications (Figure 1B), which can broadly be defined as epigenetic alterations. Thus, epigenetic mechanisms are likely to contribute to enhanceropathies, as highlighted by the fact that various cancers [43,44] and neurodevelopmental disorders [45,46] are driven by mutations in chromatin modifiers that globally deregulate chromatin and enhancer function. Another case of global chromatin deregulation was recently reported for pulmonary arterial hypertension, where widespread remodeling of acetylation of lysine 27 of H3 (H3K27ac), specifically of enhancers, seemed to prime endothelial cells to an aberrant response to transforming growth factor beta, without affecting gene expression [47]. Conceptually, this enhancer-mediated priming is similar to what has been observed in cellular differentiation (e.g., in hematopoiesis) [48,49] and may be particularly prevalent in disease mechanisms that are based on altered cell fate.
Differentially methylated regions (DMRs) between patients and controls are often enriched in enhancer loci, for example, nearly half of the 17 000 osteoarthritis DMRs are located in chondrocyte-specific enhancers ([50], more examples in Table 1). When interpreting the significance of DMRs, it is important to keep in mind that DNA methylation is highly cell type-dependent and may be driven by cell count differences. The differences between B cell methylation patterns of patients with chronic lymphocytic leukemia and healthy donors mostly reflect the cell type and maturation state from which the malignant cells originate, revealing that most ‘tumor-specific’ methylation patterns can also occur in normal development [51]. In this light, studies comparing patient-control cohorts have to carefully assess whether a change in epigenetic pattern simply reflects a change in cellular composition, or indeed is indicative of a specific enhancer activity.
Both genetic and environmental risk factors regulate the development of common multifactorial diseases and the examples in Table 1 show that enhancers can integrate genetic and environmental information in the context of disease.
Clinical implications of enhancers in disease
Insights into the disruption and misregulation of enhancers affecting disease have a wide range of implications on disease diagnosis and treatment strategies (see Clinician’s corner).
Interpreting mutations located in enhancers
Knowing the tissues in which enhancers act and the genes they regulate can improve the understanding of both common and rare disease-linked genetic variation. Next-generation sequencing in clinical settings has greatly increased the number of genetic diagnoses in patients with suspected monogenic diseases, yet many remain undiagnosed [52]. One reason for the low diagnostic yield are variants of uncertain significance that may contribute to pathogenicity, but for which we lack sufficient evidence to determine their role. Databases of variant-to-phenotype links like ClinVar [53] and variant prediction tools like CADD [54], Capice [55], and ClinPred [56] can help to annotate variants. Extending the list of disease genes to physiologically relevant enhancers and identifying their target genes will facilitate evaluating the clinical relevance of de novo mutations and their classification as benign or pathogenic.
Drug target identification
GWAS variants, which often overlap enhancers, have been used to inform drug target identification by mapping them to target genes. Two studies have shown that such genetic evidence increases the likelihood of drug targets reaching clinical practice [57,58]. However, as illustrated earlier, identifying disease genes targeted by GWAS loci remains an open challenge. Using capture Hi-C to analyze rheumatic disease loci revealed that 48 out of 454 putative disease genes were drug targets, but only 11 of them were targeted by drugs for rheumatic diseases, suggesting a strategy for repurposing existing drugs [59].
Enhancer-targeting therapies
An alternative strategy for drug discovery is to directly modulate disease-associated enhancers. A mouse model of fragile X syndrome, where mice with genetically reduced enhancer levels showed improved behavioural and neuronal features [60], supports the notion that targeting enhancers can be a successful strategy to modulate dysregulated disease genes. Some well-known chromatin-modulating drugs block the transfer of methyl and acetyl groups to histones by inhibiting DNA methyltransferase (DNMT), histone deacetylase (HDAC), histone acetyltransferase (HAT), enhancer of zeste homolog 2 (EZH2), or protein arginine N-methyltransferase (PRMT) (Box 2). One class of proteins that is of particular interest in the context of enhancers is the bromo- and extra-terminal (BET) family. BET proteins are characterized by two bromodomains that can recognize and bind to acetylated lysine residues [61] and an extra terminal domain that can interact with transcriptional activators [62]. Chromatin marks with acetylated lysines, specifically H3K27ac, are characteristic for active enhancers and BET proteins are indeed preferentially (although not exclusively) located at enhancers [63]. The family members BRD2, BRD3, and BRD4 are expressed in multiple organs, while BRDT is selectively expressed in germ cells [64] (Figure 3A). Brd4 acts as a scaffold for the transcriptional machinery and, while the molecular details are still to be discovered, the other BET proteins are expected to have a similar function [63,65]. BET proteins are at the center of a promising therapeutic direction, because their inhibition can selectively control gene expression, as reviewed in [63].
Figure 3
Figure 3 Gene expression and mechanism of bromo- and extra-terminal (BET) proteins.
Show full captionFigure viewer
Box 2
Epigenome-modulating drugs
In addition to inhibiting BET proteins (see main text), small molecules that inhibit other epigenetic regulators are gaining traction. Leukemias and myelodysplastic syndromes are already being treated with DNMT inhibitors, which prevents the transfer of methyl groups to the DNA [79]. Other studies focus on targeting histone methylation or acetylation. HDACs play a role in removing acetyl groups from histone tails. HDAC inhibitors have been recognized as potential drugs for a range of diseases, including malignancies, neurodegenerative conditions, HIV infection, and inflammatory conditions [110]. EZH2, a subunit of the polycomb repressive complex 2 that transfers methyl groups to lysine groups on histone tails, is under investigation in multiple clinical trials as a new cancer drug [79,111]. Interestingly, EZH2 inhibition has also been proposed as a treatment candidate for a number of autoimmune diseases, because of its role in autoimmune self-tolerance [112]. Similarly, PRMTs, which catalyze the transfer of methyl groups to arginine residues on histone tails, are being inhibited by a number of small molecules in clinical trials of several malignancies [78]. An overview of how BETs, DNMTs, HDACs, EHZ2, and PRMTs play a role in epigenetic modulation is presented in Figure 2 in [78].
BET inhibitors are small molecules that interact with the bromodomains of BET proteins and thereby block their interaction with acetylated histone marks (Figure 3B). Because acetylated lysines are a hallmark of active enhancers, BET inhibitors are thought to result in repressed enhancer function [63]. Although the effects of BET inhibition have mostly been studied for BRD4, the bromodomains of all four BET proteins share a high degree of homology, so BET inhibitors are expected to interact similarly with all BET proteins [63], with the exception of inhibitors that preferentially target a specific BET protein or bromodomain (Figure 3B) [65].
Currently, BET inhibitors are mostly studied as an anticancer treatment (i.e., 35 out of 45 clinical trials, see Table S1 in the supplemental information online; reviewed in detail [66]). The role of BRD3 and BRD4 in cancer was first described in nuclear protein of testis (NUT) carcinoma, where a translocation causes BRD4 to fuse to NUT, driving cancer development [66,67]. It was then shown that BET inhibition can affect enhancer-driven oncogene expression in cancer cell lines [68] and one BET inhibitor passed Phase I in a clinical trial to treat NUT carcinoma (NCT01987362i [69]). A study on prostate cancer found that selective inhibition of only the second bromodomain with ABBV-744 leads to fewer unwanted effects as compared with a pan-BET inhibitor, suggesting selective inhibition may be a valuable direction for treatment optimization [70]. BET inhibition is also under investigation for a range of other diseases. Pan-BET inhibitor I-BET762 can control the expression of secondary response genes in lipopolysaccharides-stimulated mouse macrophages, while the primary response genes and housekeeping genes are not affected [63,71]. This specificity makes the inhibition of BET proteins a particularly interesting strategy for immune-mediated diseases. Other well-studied pan-BET inhibitors, like JQ1 and I-BET151, exhibit similar inflammation-controlling behaviors in a range of cell types [63] and seem to specifically target enhancer-regulated disease genes [72].
The bromodomain-2 specific apabetalone is the most studied drug target for a range of cardiovascular phenotypes, including dyslipidemia, coronary artery disease, and type 2 diabetes [65,73,74] and is currently in a Phase I/II clinical trial for Fabry disease (a lysosomal storage disorder, NCT03228940ii) and chronic kidney failure (NCT03160430iii) and in an early Phase I for pulmonary arterial hypertension (NCT03655704iv). Nevertheless, there are still many open questions regarding BET inhibition, as illustrated by a recent study that found the timing of the JQ1 treatment impacted its effect on acute kidney injury in mice [75]. Mice that were treated with BET inhibitors soon after acute kidney injury had a higher mortality rate, while those with a later treatment showed decreased fibrosis. Perhaps the different effects can be linked to transcriptional programs that are activated in sequence, which emphasizes that understanding how and when enhancers regulate their target genes is important in understanding the clinical impact of BET inhibitors as well.
BET inhibitors are a new class of small molecule drugs and additional compounds are likely to be discovered in the near future. For example, a drug screen on keratinocytes identified and optimized a new BET inhibiting compound that improved skin healing in vivo [76]. Yet, while (super) enhancers seem to be specifically sensitive to BET inhibition [76], recent work has shown that enhancer–promoter loops stay intact upon BET treatment, despite major changes at the transcriptional level [77]. Thus, the exact role of BET proteins in transcriptional regulation is not fully understood, in part because enhancer activity upon BET inhibition is often not directly measured [63]. Taken together, BET inhibitors, as well as other epigenetic inhibitors under investigation for their therapeutic potential [78,79] (Box 2), hold a great promise for future therapeutic directions. Although these strategies do not target enhancers per se, the fact that adapting the epigenomic landscape is a viable treatment option indicates that drug target identification does not always need to focus on specific proteins or enzymes, but rather that the entire regulatory layer of the genome becomes druggable.
For diseases where enhancers drive the expression of unwanted gene products, another new treatment option is to edit the enhancer sequence using CRISPR editing techniques. This therapy has been pioneered in transfusion-dependent β-thalassemia (TDT) and sickle cell disease (SCD) [80]. The study shows that editing an enhancer in the patients’ hematopoietic stem and progenitor cells donor cells, myelo-ablating the patients, and injecting them with the edited cells, resulted in the patients no longer needing transfusions and having vaso-occlusive episodes. A handful of Phase I/II clinical trials are currently recruiting to test this strategy in a larger group of TDT and SCD patients (NCT03655678v, NCT03745287vi, NCT04443907vii). Although there are currently few trials that use CRISPR-Cas9 edited cells as treatment and even fewer that target enhancers, CRISPR technology has had an enormous impact on the characterization of enhancers [41,42,81] and it is becoming possible to directly edit the genome in vivo, as pioneered in a small cohort of patients suffering from transthyretin amyloidosis [82]. Consequently, it is likely that therapies that directly knockout or edit specific enhancers in a subset of cells will become a topic of intense research in the next few years.
Concluding remarks
Although enhancers are increasingly recognized as important drivers of disease-related processes, key challenges to comprehensively understand the fundamental biological mechanisms and their role in disease remain (see Outstanding questions). The exact definition of what an enhancer is depends on the technology used to identify them [83]; the identification of genes regulated by an enhancer is not as straightforward as finding the nearest gene and enhancers tend to be highly specific to a limited number of tissues, cell types, or developmental stages [84–86].
Despite these challenges, a new class of treatments that aims to inhibit enhancers directly has shown great promise in cancer and immune-related diseases. Moreover, pioneering approaches in gene editing have proven the feasibility of editing one specific enhancer to correct a disease-causing mutation. Overall, a better understanding of enhancer function, including enhancer maps in specific cell types, developmental stages, and disease states, as well as their interaction with cell-extrinsic signaling pathways, will pave the way for novel therapy approaches.
Clinician’s corner
Numerous cancers and congenital conditions arise from disruption of enhancers, the genomic sequences that regulate the expression of genes. The majority of genetic loci linked to common diseases are located in putative enhancer regions, suggesting their importance in both rare and multifactorial diseases. The number of enhancer-driven mechanisms in these diseases is likely underestimated given the challenges of enhancer identification and characterization. Whole-genome sequencing and epigenetic profiling (e.g., chromatin accessibility) in the relevant disease tissue in a clinical context will improve our understanding of enhanceropathies.
Overlapping tissue- and developmental stage-specific maps of enhancers with putative disease-causing mutations can reveal novel enhancer-mediated disease mechanisms and may help to interpret genetic variants of unknown significance.
Enhancers can be targeted using BET inhibitors, small molecules that inhibit enhancer-driven gene expression. BET inhibitors are currently in clinical trials as treatment for cancers, cardiovascular diseases, and autoimmune diseases, but the exact mechanism of action is still under investigation.
Recent approaches in gene editing using CRISPR offer more precise enhancer targeting. CRISPR can be used to edit, inhibit, or activate a specific enhancer sequence in donor cells ex vivo, which can then be injected into a patient. This approach is particularly appealing for blood disorders, because patients can be myelo-ablated and the donor stem cells can replace the diseased cells. In the future, it may also be possible to repair the patient’s own cells.
Outstanding questions
What are the genes targeted by specific enhancers that overlap with disease-associated genetic variants? Addressing this question will be important to understand disease mechanisms for many common diseases. In silico prediction methods along with high-throughput experimental enhancer validation approaches will be key to map enhancers to their target genes.
In which tissues and in what developmental or perturbation context do disease-associated enhancers function? Context-specific enhancer annotations will be required to answer this question.
If an enhancer is functional in multiple tissues or developmental stages, are the target genes the same across these contexts? Importantly, enhancers are often part of a regulatory cluster, so combinatorial perturbations may be required to understand the redundancy in gene regulation.
To what extent do genetic variants/mutations in developmental enhancers contribute to disease? Chromatin profiling in single cells across developmental stages will be crucial to identify developmental enhancers.
Is the disruption of one or more TAD boundaries a common disease mechanism? To investigate this, Hi-C and chromatin-capture technologies in clinical settings will be key.
Is enhancer-hijacking a common driver event in cancers, or is it a rare disease mechanism? Whole-genome sequencing and chromatin profiling of cancer tissue will be crucial to address this question. In case it is a common driver event, the question arises whether such hijacking mechanisms can specifically be targeted.
How do BET inhibitors achieve specificity on their target genes given that Brd proteins bind the chromatin very globally? Addressing this question will help understand whether some tissues may be more responsive to BET inhibition and thus help determine what diseases may benefit from BET inhibition.
Do BET inhibitors that target only one domain give more favorable clinical outcomes? Having access to BET inhibitors with variable strength will allow nuanced dosing of medication.
Acknowledgments
Acknowledgements
The authors would like to thank Remi Terranova, Jorge Ferrer, Pelin Sahlen, Susanne Mandrup, Salvatore Spicuglia, Bart Deplancke, and Wouter de Laat for their helpful suggestions for this review.
Declaration of interests
No interests are declared.
Supplemental information (1)
Spreadsheet (73.89 KB)
Supplementary Table S1
Overview of the clinical trials investigating BET inhibitors as reported on clinicaltrials.gov, accessed on 07-05-2021.
Resources
iwww.clinicaltrials.gov/ct2/show/NCT01987362
iiwww.clinicaltrials.gov/ct2/show/NCT03228940
iiiwww.clinicaltrials.gov/ct2/show/NCT03160430
ivwww.clinicaltrials.gov/ct2/show/NCT03655704
vwww.clinicaltrials.gov/ct2/show/NCT03655678
viwww.clinicaltrials.gov/ct2/show/NCT03745287
viiwww.clinicaltrials.gov/ct2/show/NCT04443907
References
1.
Pott, S. ∙ Lieb, J.D.
What are super-enhancers?
Nat. Genet. 2015; 47:8-12
Crossref
Scopus (330)
PubMed
Google Scholar
2.
Hnisz, D. ...
A phase separation model for transcriptional control
Cell. 2017; 169:13-23
Full Text
Full Text (PDF)
Scopus (615)
PubMed
Google Scholar
3.
Banerji, J. ...
Expression of a beta-globin gene is enhanced by remote SV40 DNA sequences
Cell. 1981; 27:299-308
Abstract
Full Text (PDF)
Scopus (849)
PubMed
Google Scholar
4.
Kioussis, D. ...
β-Globin gene inactivation by DNA translocation in γβ-thalassaemi
Nature. 1983; 306:662-666
Crossref
Scopus (0)
PubMed
Google Scholar
5.
Smith, E. ∙ Shilatifard, A.
Enhancer biology and enhanceropathies
Nat. Struct. Mol. Biol. 2014; 21:210-219
Crossref
Scopus (162)
PubMed
Google Scholar
6.
Lettice, L.A. ...
Disruption of a long-range cis-acting regulator for Shh causes preaxial polydactyly
Proc. Natl. Acad. Sci. U. S. A. 2002; 99:7548-7553
Crossref
Scopus (0)
PubMed
Google Scholar
7.
Taub, R. ...
Translocation of the c-myc gene into the immunoglobulin heavy chain locus in human Burkitt lymphoma and murine plasmacytoma cells
Proc. Natl. Acad. Sci. U. S. A. 1982; 79:7837-7841
Crossref
Scopus (0)
PubMed
Google Scholar
8.
Northcott, P.A. ...
Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma
Nature. 2014; 511:428-434
Crossref
Scopus (312)
PubMed
Google Scholar
9.
Abraham, B.J. ...
Small genomic insertions form enhancers that misregulate oncogenes
Nat. Commun. 2017; 8:14385
Crossref
Scopus (38)
PubMed
Google Scholar
10.
Zhang, Y. ...
High-coverage whole-genome analysis of 1220 cancers reveals hundreds of genes deregulated by rearrangement-mediated cis-regulatory alterations
Nat. Commun. 2020; 11:736
Crossref
Scopus (21)
PubMed
Google Scholar
11.
Schoenfelder, S. ∙ Fraser, P.
Long-range enhancer-promoter contacts in gene expression control
Nat. Rev. Genet. 2019; 20:437-455
Crossref
Scopus (226)
PubMed
Google Scholar
12.
Lupiáñez, D.G. ...
Disruptions of topological chromatin domains cause pathogenic rewiring of gene-enhancer interactions
Cell. 2015; 161:1012-1025
Full Text
Full Text (PDF)
Scopus (941)
PubMed
Google Scholar
13.
Sun, J.H. ...
Disease-associated short tandem repeats co-localize with chromatin domain boundaries
Cell. 2018; 175:224-238
Full Text
Full Text (PDF)
PubMed
Google Scholar
14.
Swaminathan, G.J. ...
DECIPHER: web-based, community resource for clinical interpretation of rare variants in developmental disorders
Hum. Mol. Genet. 2012; 21:R37-R44
Crossref
Scopus (0)
PubMed
Google Scholar
15.
Kaiser, V.B. ∙ Semple, C.A.
When TADs go bad: chromatin structure and nuclear organisation in human disease
F1000Res. 2017; 6:314
Crossref
Scopus (1)
Google Scholar
16.
Akdemir, K.C. ...
Disruption of chromatin folding domains by somatic genomic rearrangements in human cancer
Nat. Genet. 2020; 52:294-305
Crossref
Scopus (43)
PubMed
Google Scholar
17.
Smol, T. ...
Lessons from the analysis of TAD boundary deletions in normal population
BioRxiv. 2020;
Published online April 3, 2020. https://doi.org/10.1101/2020.04.01.021188
Google Scholar
18.
Wang, X. ∙ Goldstein, D.B.
Enhancer domains predict gene pathogenicity and inform gene discovery in complex disease
Am. J. Hum. Genet. 2020; 106:215-233
Full Text
Full Text (PDF)
Scopus (10)
PubMed
Google Scholar
19.
Karnuta, J.M. ∙ Scacheri, P.C.
Enhancers: bridging the gap between gene control and human disease
Hum. Mol. Genet. 2018; 27:R219-R227
Crossref
PubMed
Google Scholar
20.
Carullo, N.V.N. ∙ Day, J.J.
Genomic enhancers in brain health and disease
Genes (Basel). 2019; 10:43
Crossref
Scopus (12)
PubMed
Google Scholar
21.
De Vas, M.G. ...
De novo mutations in fetal brain specific enhancers play a significant role in severe intellectual disability
BioRxiv. 2019;
Published online April 28, 2019 https://doi.org/10.1101/621029
Google Scholar
22.
Domcke, S. ...
A human cell atlas of fetal chromatin accessibility
Science. 2020; 370, eaba7612
Crossref
Scopus (32)
PubMed
Google Scholar
23.
Zhang, K. ...
A cell atlas of chromatin accessibility across 25 adult human tissues
BioRxiv. 2021;
Published online February 17, 2021 https://doi.org/10.1101/2021.02.17.431699
Google Scholar
24.
Boix, C.A. ...
Regulatory genomic circuitry of human disease loci by integrative epigenomics
Nature. 2021; 590:300-307
Crossref
Scopus (20)
PubMed
Google Scholar
25.
Gjoneska, E. ...
Conserved epigenomic signals in mice and humans reveal immune basis of Alzheimer’s disease
Nature. 2015; 518:365-369
Crossref
Scopus (319)
PubMed
Google Scholar
26.
Schwartzentruber, J. ...
Genome-wide meta-analysis, fine-mapping and integrative prioritization implicate new Alzheimer’s disease risk genes
Nat. Genet. 2021; 53:392-402
Crossref
Scopus (8)
PubMed
Google Scholar
27.
Giambartolomei, C. ...
Bayesian test for colocalisation between pairs of genetic association studies using summary statistics
PLoS Genet. 2014; 10, e1004383
Crossref
Scopus (528)
PubMed
Google Scholar
28.
GTEx Consortium
The GTEx Consortium atlas of genetic regulatory effects across human tissues
Science. 2020; 369:1318-1330
Crossref
PubMed
Google Scholar
29.
Nasser, J. ...
Genome-wide maps of enhancer regulation connect risk variants to disease genes
BioRxiv. 2021; 593:238-243
Google Scholar
30.
Yao, D.W. ...
Quantifying genetic effects on disease mediated by assayed gene expression levels
Nat. Genet. 2020; 52:626-633
Crossref
Scopus (25)
PubMed
Google Scholar
31.
Zeng, J. ...
Signatures of negative selection in the genetic architecture of human complex traits
Nat. Genet. 2018; 50:746-753
Crossref
Scopus (89)
PubMed
Google Scholar
32.
Võsa, U. ...
Unraveling the polygenic architecture of complex traits using blood eQTL meta-analysis
BioRxiv. 2018;
Published online October 19, 2018 https://doi.org/10.1101/447367
Google Scholar
33.
Broekema, R.V. ...
A practical view of fine-mapping and gene prioritization in the post-genome-wide association era
Open Biol. 2020; 10:190221
Crossref
Scopus (13)
PubMed
Google Scholar
34.
Lu, L. ...
Robust Hi-C maps of enhancer-promoter interactions reveal the function of non-coding genome in neural development and diseases
Mol. Cell. 2020; 79:521-534
Full Text
Full Text (PDF)
Scopus (25)
PubMed
Google Scholar
35.
Ron, G. ...
Promoter-enhancer interactions identified from Hi-C data using probabilistic models and hierarchical topological domains
Nat. Commun. 2017; 8:2237
Crossref
Scopus (52)
PubMed
Google Scholar
36.
Miguel-Escalada, I. ...
Human pancreatic islet three-dimensional chromatin architecture provides insights into the genetics of type 2 diabetes
Nat. Genet. 2019; 51:1137-1148
Crossref
Scopus (70)
PubMed
Google Scholar
37.
Warren, H.R. ...
Genome-wide association analysis identifies novel blood pressure loci and offers biological insights into cardiovascular risk
Nat. Genet. 2017; 49:403-415
Crossref
Scopus (247)
PubMed
Google Scholar
38.
McCord, R.P. ...
Chromosome conformation capture and beyond: toward an integrative view of chromosome structure and function
Mol. Cell. 2020; 77:688-708
Full Text
Full Text (PDF)
Scopus (29)
PubMed
Google Scholar
39.
Chatterjee, S. ...
Enhancer variants synergistically drive dysfunction of a gene regulatory network in Hirschsprung disease
Cell. 2016; 167:355-368
Full Text
Full Text (PDF)
Scopus (68)
PubMed
Google Scholar
40.
Boyle, E.A. ...
An expanded view of complex traits: from polygenic to omnigenic
Cell. 2017; 169:1177-1186
Full Text
Full Text (PDF)
Scopus (1031)
PubMed
Google Scholar
41.
Schraivogel, D. ...
Targeted Perturb-seq enables genome-scale genetic screens in single cells
Nat. Methods. 2020; 17:629-635
Crossref
Scopus (17)
PubMed
Google Scholar
42.
Dixit, A. ...
Perturb-seq: dissecting molecular circuits with scalable single-cell RNA profiling of pooled genetic screens
Cell. 2016; 167:1853-1866
Full Text
Full Text (PDF)
Scopus (464)
PubMed
Google Scholar
43.
Valencia, A.M. ∙ Kadoch, C.
Chromatin regulatory mechanisms and therapeutic opportunities in cancer
Nat. Cell Biol. 2019; 21:152-161
Crossref
Scopus (52)
PubMed
Google Scholar
44.
Mirabella, A.C. ...
Chromatin deregulation in disease
Chromosoma. 2016; 125:75-93
Crossref
Scopus (50)
PubMed
Google Scholar
45.
Vitriolo, A. ...
From enhanceropathies to the epigenetic manifold underlying human cognition
Hum. Mol. Genet. 2019; 28:R226-R234
Crossref
Scopus (3)
PubMed
Google Scholar
46.
Gabriele, M. ...
The chromatin basis of neurodevelopmental disorders: rethinking dysfunction along the molecular and temporal axes
Prog. Neuro-Psychopharmacol. Biol. Psychiatry. 2018; 84:306-327
Crossref
Scopus (27)
PubMed
Google Scholar
47.
Reyes-Palomares, A. ...
Remodeling of active endothelial enhancers is associated with aberrant gene-regulatory networks in pulmonary arterial hypertension
Nat. Commun. 2020; 11:1673
Crossref
Scopus (0)
PubMed
Google Scholar
48.
Ranzoni, A.M. ...
Integrative single-cell RNA-seq and ATAC-seq analysis of human developmental hematopoiesis
Cell Stem Cell. 2021; 28:472-487
Full Text
Full Text (PDF)
Scopus (13)
PubMed
Google Scholar
49.
Ma, S. ...
Chromatin potential identified by shared single-cell profiling of RNA and chromatin
Cell. 2020; 183:1103-1116
Full Text
Full Text (PDF)
Scopus (46)
PubMed
Google Scholar
50.
Lin, X. ...
Genome-wide analysis of aberrant methylation of enhancer DNA in human osteoarthritis
BMC Med. Genet. 2020; 13:1
Google Scholar
51.
Oakes, C.C. ...
DNA methylation dynamics during B cell maturation underlie a continuum of disease phenotypes in chronic lymphocytic leukemia
Nat. Genet. 2016; 48:253-264
Crossref
Scopus (165)
PubMed
Google Scholar
52.
Wright, C.F. ...
Paediatric genomics: diagnosing rare disease in children
Nat. Rev. Genet. 2018; 19:253-268
Crossref
Scopus (171)
PubMed
Google Scholar
53.
Landrum, M.J. ...
ClinVar: public archive of relationships among sequence variation and human phenotype
Nucleic Acids Res. 2014; 42:D980-D985
Crossref
Scopus (1337)
PubMed
Google Scholar
54.
Rentzsch, P. ...
CADD: predicting the deleteriousness of variants throughout the human genome
Nucleic Acids Res. 2019; 47:D886-D894
Crossref
Scopus (809)
PubMed
Google Scholar
55.
Li, S. ...
CAPICE: a computational method for consequence-agnostic pathogenicity interpretation of clinical exome variations
Genome Med. 2020; 12:75
Crossref
Scopus (3)
PubMed
Google Scholar
56.
Alirezaie, N. ...
ClinPred: prediction tool to identify disease-relevant nonsynonymous single-nucleotide variants
Am. J. Hum. Genet. 2018; 103:474-483
Full Text
Full Text (PDF)
Scopus (43)
PubMed
Google Scholar
57.
Cook, D. ...
Lessons learned from the fate of AstraZeneca’s drug pipeline: a five-dimensional framework
Nat. Rev. Drug Discov. 2014; 13:419-431
Crossref
Scopus (735)
PubMed
Google Scholar
58.
Nelson, M.R. ...
The support of human genetic evidence for approved drug indications
Nat. Genet. 2015; 47:856-860
Crossref
Scopus (525)
PubMed
Google Scholar
59.
Martin, P. ...
Chromatin interactions reveal novel gene targets for drug repositioning in rheumatic diseases
Ann. Rheum. Dis. 2019; 78:1127-1134
Crossref
Scopus (15)
PubMed
Google Scholar
60.
Gross, C. ...
Increased expression of the PI3K enhancer PIKE mediates deficits in synaptic plasticity and behavior in fragile X syndrome
Cell Rep. 2015; 11:727-736
Full Text
Full Text (PDF)
Scopus (68)
PubMed
Google Scholar
61.
Zeng, L. ∙ Zhou, M.M.
Bromodomain: an acetyl-lysine binding domain
FEBS Lett. 2002; 513:124-128
Crossref
Scopus (504)
PubMed
Google Scholar
62.
Rahman, S. ...
The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3
Mol. Cell. Biol. 2011; 31:2641-2652
Crossref
Scopus (314)
PubMed
Google Scholar
63.
Tough, D.F. ∙ Prinjha, R.K.
Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention
Epigenomics. 2017; 9:573-584
Crossref
Scopus (23)
PubMed
Google Scholar
64.
Wang, N. ...
The BET family in immunity and disease
Signal. Transduct. Target. Ther. 2021; 6:23
Crossref
Scopus (7)
PubMed
Google Scholar
65.
Borck, P.C. ...
BET epigenetic reader proteins in cardiovascular transcriptional programs
Circ. Res. 2020; 126:1190-1208
Crossref
Scopus (21)
PubMed
Google Scholar
66.
Shorstova, T. ...
Achieving clinical success with BET inhibitors as anti-cancer agents
Br. J. Cancer. 2021; 124:1478-1490
Crossref
Scopus (4)
PubMed
Google Scholar
67.
French, C.A. ...
BRD4-NUT fusion oncogene: a novel mechanism in aggressive carcinoma
Cancer Res. 2003; 63:304-307
PubMed
Google Scholar
68.
Lovén, J. ...
Selective inhibition of tumor oncogenes by disruption of super-enhancers
Cell. 2013; 153:320-334
Full Text
Full Text (PDF)
Scopus (1492)
PubMed
Google Scholar
69.
Shapiro, G.I. ...
A Phase 1 study of RO6870810, a novel bromodomain and extra-terminal protein inhibitor, in patients with NUT carcinoma, other solid tumours, or diffuse large B-cell lymphoma
Br. J. Cancer. 2021; 124:744-753
Crossref
Scopus (2)
PubMed
Google Scholar
70.
Faivre, E.J. ...
Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer
Nature. 2020; 578:306-310
Crossref
Scopus (92)
PubMed
Google Scholar
71.
Nicodeme, E. ...
Suppression of inflammation by a synthetic histone mimic
Nature. 2010; 468:1119-1123
Crossref
Scopus (1071)
PubMed
Google Scholar
72.
Peeters, J.G.C. ...
Inhibition of super-enhancer activity in autoinflammatory site-derived T cells reduces disease-associated gene expression
Cell Rep. 2015; 12:1986-1996
Full Text
Full Text (PDF)
PubMed
Google Scholar
73.
Ray, K.K. ...
Effect of selective BET protein inhibitor apabetalone on cardiovascular outcomes in patients with acute coronary syndrome and diabetes: rationale, design, and baseline characteristics of the BETonMACE trial
Am. Heart J. 2019; 217:72-83
Crossref
Scopus (16)
PubMed
Google Scholar
74.
Wasiak, S. ...
BET protein inhibitor apabetalone (RVX-208) suppresses pro-inflammatory hyper-activation of monocytes from patients with cardiovascular disease and type 2 diabetes
Clin. Epigenetics. 2020; 12:166
Crossref
Scopus (2)
PubMed
Google Scholar
75.
Wilflingseder, J. ...
Enhancer and super-enhancer dynamics in repair after ischemic acute kidney injury
Nat. Commun. 2020; 11:3383
Crossref
Scopus (5)
PubMed
Google Scholar
76.
Schutzius, G. ...
BET bromodomain inhibitors regulate keratinocyte plasticity
Nat. Chem. Biol. 2021; 17:280-290
Crossref
Scopus (0)
PubMed
Google Scholar
77.
Crump, N.T. ...
BET inhibition disrupts transcription but retains enhancer-promoter contact
Nat. Commun. 2021; 12:223
Crossref
Scopus (6)
PubMed
Google Scholar
78.
Sermer, D. ...
Emerging epigenetic-modulating therapies in lymphoma
Nat. Rev. Clin. Oncol. 2019; 16:494-507
Crossref
Scopus (44)
PubMed
Google Scholar
79.
Genta, S. ...
BET and EZH2 inhibitors: novel approaches for targeting cancer
Curr. Oncol. Rep. 2019; 21:13
Crossref
Scopus (17)
PubMed
Google Scholar
80.
Frangoul, H. ...
CRISPR-Cas9 gene editing for sickle cell disease and β-thalassemia
N. Engl. J. Med. 2021; 384:252-260
Crossref
Scopus (110)
PubMed
Google Scholar
81.
Li, K. ...
Interrogation of enhancer function by enhancer-targeting CRISPR epigenetic editing
Nat. Commun. 2020; 11:485
Crossref
Scopus (14)
PubMed
Google Scholar
82.
Gillmore, J.D. ...
CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis
N. Engl. J. Med. 2021; 385:493-502
Crossref
Scopus (9)
PubMed
Google Scholar
83.
Rickels, R. ∙ Shilatifard, A.
Enhancer logic and mechanics in development and disease
Trends Cell Biol. 2018; 28:608-630
Full Text
Full Text (PDF)
Scopus (71)
PubMed
Google Scholar
84.
Andersson, R. ...
An atlas of active enhancers across human cell types and tissues
Nature. 2014; 507:455-461
Crossref
Scopus (1282)
PubMed
Google Scholar
85.
Gao, T. ∙ Qian, J.
EnhancerAtlas 2.0: an updated resource with enhancer annotation in 586 tissue/cell types across nine species
Nucleic Acids Res. 2020; 48:D58-D64
PubMed
Google Scholar
86.
Roadmap Epigenomics Consortium et al
Integrative analysis of 111 reference human epigenomes
Nature. 2015; 518:317-330
Crossref
Scopus (2843)
PubMed
Google Scholar
87.
Spitz, F. ∙ Furlong, E.E.M.
Transcription factors: from enhancer binding to developmental control
Nat. Rev. Genet. 2012; 13:613-626
Crossref
Scopus (961)
PubMed
Google Scholar
88.
Dixon, J.R. ...
Topological domains in mammalian genomes identified by analysis of chromatin interactions
Nature. 2012; 485:376-380
Crossref
Scopus (3314)
PubMed
Google Scholar
89.
Krijger, P.H.L. ∙ de Laat, W.
Regulation of disease-associated gene expression in the 3D genome
Nat. Rev. Mol. Cell Biol. 2016; 17:771-782
Crossref
Scopus (124)
PubMed
Google Scholar
90.
Nora, E.P. ...
Spatial partitioning of the regulatory landscape of the X-inactivation centre
Nature. 2012; 485:381-385
Crossref
Scopus (1527)
PubMed
Google Scholar
91.
Rao, S.S.P. ...
A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping
Cell. 2014; 159:1665-1680
Full Text
Full Text (PDF)
Scopus (2775)
PubMed
Google Scholar
92.
Rao, S.S.P. ...
Cohesin loss eliminates all loop domains
Cell. 2017; 171:305-320
Full Text
Full Text (PDF)
Scopus (629)
PubMed
Google Scholar
93.
Schwarzer, W. ...
Two independent modes of chromatin organization revealed by cohesin removal
Nature. 2017; 551:51-56
Crossref
Scopus (426)
PubMed
Google Scholar
94.
Furlong, E.E.M. ∙ Levine, M.
Developmental enhancers and chromosome topology
Science. 2018; 361:1341-1345
Crossref
Scopus (168)
PubMed
Google Scholar
95.
Ruiz-Velasco, M. ∙ Zaugg, J.B.
Structure meets function: how chromatin organisation conveys functionality
Curr. Opin. Syst. Biol. 2017; 1:129-136
Crossref
Google Scholar
96.
Grubert, F. ...
Landscape of cohesin-mediated chromatin loops in the human genome
Nature. 2020; 583:737-743
Crossref
Scopus (23)
PubMed
Google Scholar
97.
Liu, N.Q. ...
WAPL maintains a cohesin loading cycle to preserve cell-type-specific distal gene regulation
Nat. Genet. 2021; 53:100-109
Crossref
Scopus (13)
PubMed
Google Scholar
98.
Ghavi-Helm, Y. ...
Highly rearranged chromosomes reveal uncoupling between genome topology and gene expression
Nat. Genet. 2019; 51:1272-1282
Crossref
Scopus (97)
PubMed
Google Scholar
99.
Ing-Simmons, E. ...
Independence of 3D chromatin conformation and gene regulation during Drosophila dorsoventral patterning
Nat. Genet. 2021; 53:487-499
Crossref
Scopus (0)
PubMed
Google Scholar
100.
Arnold, C.D. ...
Genome-wide quantitative enhancer activity maps identified by STARR-seq
Science. 2013; 339:1074-1077
Crossref
Scopus (477)
PubMed
Google Scholar
101.
Inoue, F. ...
A systematic comparison reveals substantial differences in chromosomal versus episomal encoding of enhancer activity
Genome Res. 2017; 27:38-52
Crossref
Scopus (100)
PubMed
Google Scholar
102.
Fulco, C.P. ...
Activity-by-contact model of enhancer-promoter regulation from thousands of CRISPR perturbations
Nat. Genet. 2019; 51:1664-1669
Crossref
Scopus (110)
PubMed
Google Scholar
103.
Spicuglia, S. ∙ Vanhille, L.
Chromatin signatures of active enhancers
Nucleus. 2012; 3:126-131
Crossref
PubMed
Google Scholar
104.
Ong, C.-T. ∙ Corces, V.G.
Enhancer function: new insights into the regulation of tissue-specific gene expression
Nat. Rev. Genet. 2011; 12:283-293
Crossref
Scopus (533)
PubMed
Google Scholar
105.
Ernst, J. ∙ Kellis, M.
ChromHMM: automating chromatin-state discovery and characterization
Nat. Methods. 2012; 9:215-216
Crossref
Scopus (1098)
PubMed
Google Scholar
106.
Kim, T.-K. ...
Enhancer RNAs: a class of long noncoding RNAs synthesized at enhancers
Cold Spring Harb. Perspect. Biol. 2015; 7, a018622
Crossref
Scopus (82)
PubMed
Google Scholar
107.
Miguel-Escalada, I. ...
Transcriptional enhancers: functional insights and role in human disease
Curr. Opin. Genet. Dev. 2015; 33:71-76
Crossref
Scopus (16)
PubMed
Google Scholar
108.
Bravo González-Blas, C. ...
Identification of genomic enhancers through spatial integration of single-cell transcriptomics and epigenomics
Mol. Syst. Biol. 2020; 16, e9438
Crossref
Scopus (12)
PubMed
Google Scholar
109.
Boija, A. ...
Transcription factors activate genes through the phase-separation capacity of their activation domains
Cell. 2018; 175:1842-1855
Full Text
Full Text (PDF)
Scopus (428)
PubMed
Google Scholar
110.
Bondarev, A.D. ...
Recent developments of HDAC inhibitors: emerging indications and novel molecules
Br. J. Clin. Pharmacol. 2021;
Published online May 10, 2021 https://doi.org/10.1111/bcp.14889
Scopus (1)
PubMed
Google Scholar
111.
Gan, L. ...
Epigenetic regulation of cancer progression by EZH2: from biological insights to therapeutic potential
Biomark. Res. 2018; 6:10
Crossref
Scopus (201)
PubMed
Google Scholar
112.
Yang, Y.-X. ...
Therapeutic potential of enhancer of zeste homolog 2 in autoimmune diseases
Expert Opin. Ther. Targets. 2019; 23:1015-1030
Crossref
Scopus (3)
PubMed
Google Scholar
113.
Helmsauer, K. ...
Enhancer hijacking determines extrachromosomal circular MYCN amplicon architecture in neuroblastoma
Nat. Commun. 2020; 11:5823
Crossref
Scopus (7)
PubMed
Google Scholar
114.
Haller, F. ...
Enhancer hijacking activates oncogenic transcription factor NR4A3 in acinic cell carcinomas of the salivary glands
Nat. Commun. 2019; 10:368
Crossref
Scopus (61)
PubMed
Google Scholar
115.
Weischenfeldt, J. ...
Pan-cancer analysis of somatic copy-number alterations implicates IRS4 and IGF2 in enhancer hijacking
Nat. Genet. 2017; 49:65-74
Crossref
Scopus (170)
PubMed
Google Scholar
116.
Ooi, W.F. ...
Integrated paired-end enhancer profiling and whole-genome sequencing reveals recurrent CCNE1 and IGF2 enhancer hijacking in primary gastric adenocarcinoma
Gut. 2020; 69:1039-1052
Crossref
Scopus (9)
PubMed
Google Scholar
117.
Yang, L. ...
3D genome analysis identifies enhancer hijacking mechanism for high-risk factors in human T-lineage acute lymphoblastic leukemia
BioRxiv. 2020;
Published online Mach 12, 2020 https://doi.org/10.1101/2020.03.11.988279
Google Scholar
118.
Redin, C. ...
The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies
Nat. Genet. 2017; 49:36-45
Crossref
PubMed
Google Scholar
119.
Benko, S. ...
Highly conserved non-coding elements on either side of SOX9 associated with Pierre Robin sequence
Nat. Genet. 2009; 41:359-364
Crossref
Scopus (266)
PubMed
Google Scholar
120.
Smemo, S. ...
Regulatory variation in a TBX5 enhancer leads to isolated congenital heart disease
Hum. Mol. Genet. 2012; 21:3255-3263
Crossref
Scopus (117)
PubMed
Google Scholar
121.
Weedon, M.N. ...
Recessive mutations in a distal PTF1A enhancer cause isolated pancreatic agenesis
Nat. Genet. 2014; 46:61-64
Crossref
Scopus (166)
PubMed
Google Scholar
122.
Maurano, M.T. ...
Systematic localization of common disease-associated variation in regulatory DNA
Science. 2012; 337:1190-1195
Crossref
Scopus (1901)
PubMed
Google Scholar
123.
Farh, K.K-H. ...
Genetic and epigenetic fine mapping of causal autoimmune disease variants
Nature. 2015; 518:337-343
Crossref
Scopus (964)
PubMed
Google Scholar
124.
Onengut-Gumuscu, S. ...
Fine mapping of type 1 diabetes susceptibility loci and evidence for colocalization of causal variants with lymphoid gene enhancers
Nat. Genet. 2015; 47:381-386
Crossref
Scopus (317)
PubMed
Google Scholar
125.
Pasquali, L. ...
Pancreatic islet enhancer clusters enriched in type 2 diabetes risk-associated variants
Nat. Genet. 2014; 46:136-143
Crossref
Scopus (303)
PubMed
Google Scholar
126.
Baars, M.J.D. ...
Dysregulated RASGRP1 expression through RUNX1 mediated transcription promotes autoimmunity
Eur. J. Immunol. 2020; 51:471-482
Crossref
Scopus (0)
PubMed
Google Scholar
127.
Li, P. ...
Epigenetic dysregulation of enhancers in neurons is associated with Alzheimer’s disease pathology and cognitive symptoms
Nat. Commun. 2019; 10:2246
Crossref
Scopus (62)
PubMed
Google Scholar
128.
Mordaunt, C.E. ...
Epigenomic signatures in liver and blood of Wilson disease patients include hypermethylation of liver-specific enhancers
Epigenetics Chromatin. 2019; 12:10
Crossref
Scopus (16)
PubMed
Google Scholar
129.
Souren, N.Y. ...
DNA methylation signatures of monozygotic twins clinically discordant for multiple sclerosis
Nat. Commun. 2019; 10:2094
Crossref
Scopus (0)
PubMed
Google Scholar
130.
Sheffield, N.C. ...
DNA methylation heterogeneity defines a disease spectrum in Ewing sarcoma
Nat. Med. 2017; 23:386-395
Crossref
Scopus (97)
PubMed
Google Scholar
Glossary
Acetylation of lysine 27 of H3 (H3K27ac)
modification marking active chromatin.
Bromo- and extra-terminal(BET)
protein that can recognize and bind to acetylated lysine residues and interact with transcriptional activators.
BET inhibitor
small molecule that can inhibit BET proteins from interacting with acetylated lysine residues.
Copy number variation (CNV)
a region in the genome that is repeated a variable number of times with respect to a reference genome.
CRISPR editing
gene editing that uses CRISPR (clustered regularly interspaced short palindromic repeats) and guiding sequences to edit a specific location in the genome.
Differentially methylated region (DMR)
a region in the genome, often a CpG site or CpG island, that is differentially methylated between two conditions (e.g., healthy versus diseased).
DNA methyltransferase (DNMT)
enzyme that transfers methyl groups to the DNA.
Enhancer
regulatory sequence in the genome that affects gene expression of a nearby gene by forming a loop with its promoter.
Enhancer of zeste homolog 2 (EZH2)
a subunit of the polycomb repressive complex 2 that transfers methyl groups to lysine groups on histone tails.
Enhanceropathy
disease that results from enhancer disruption.
Expression quantitative trait locus (eQTL)
a genetic variant that is correlated to gene expression levels.
Fine-mapped GWAS
a technique that leverages linkage disequilibrium to identify the likely causal genetic variants among all GWAS associated variants.
Genome-wide association studies (GWAS)
a study linking genetic variants to a common trait or disease to identify genetic drivers of the phenotype.
Histone deacetylase (HDAC)
enzyme that removes acetyl groups from histone tails.
Histone acetyltransferase (HAT)
enzyme that adds acetyl groups onto lysine residues.
H3K4me1
(monomethylation of lysine 4 in histone H3); modification marking enhancers.
Polygenic risk score (PRS)
score that represents the combined genetic risk for a particular disease, calculated by adding GWAS effect sizes for all disease-associated loci.
Protein arginine N-methyltransferase (PRMT)
enzyme that catalyzes the transfer of methyl groups to arginine residues on histone tails.
Short tandem repeat
short sequences (up to seven base pairs) that are repeated multiple times in the genome; the number of repeats differ among individuals, so these repeats are often used to identification in a forensic context.
Stretch enhancer
a large (>3 kb) genomic region enriched for multiple chromatin marks; more common than super enhancers in the original definition (see later).
Super enhancer
a group of putative enhancers in close physical proximity, usually assumed to coregulate a (group of) gene(s); originally defined as having a higher level of mediator binding as compared with background levels.
Transcription factor (TF)
protein that controls transcription in the nucleus.
Topologically associated domain (TAD)
genomic region that has a high level of interaction within and no interactions outside, relatively conserved across cell types.
Variant of uncertain significance
rare genetic variant whose clinical impact is uncertain, often found while whole-genome sequencing patients with a rare disease.
Figures (3)
Figure Viewer



Article metrics
110
Citations
254
Captures
1
Mentions
21
Social Media
Plum X logo
View details
opens in a new tab
Supplemental information (1)
Spreadsheet (73.89 KB)
Supplementary Table S1
Related articles (40)
Molecular Basis of Disease Resistance and Perspectives on Breeding Strategies for Resistance Improvement in Crops
Deng et al.
Molecular PlantSeptember 22, 2020
Emerging Molecular Targets for Treatment of Nonalcoholic Fatty Liver Disease
Chen et al.
Trends in Endocrinology & MetabolismOctober 6, 2019
The Molecular Basis of Cystathionine β-Synthase Deficiency in Dutch Patients with Homocystinuria: Effect of CBS Genotype on Biochemical and Clinical Phenotype and on Response to Treatment
Kluijtmans et al.
The American Journal of Human GeneticsJuly, 1999
Show more
Advertisement

      
Life & medical sciences journals
Cell
Cancer Cell
Cell Chemical Biology
Cell Genomics
Cell Host & Microbe
Cell Metabolism
Cell Reports
Cell Reports Medicine
Cell Stem Cell
Cell Systems
Current Biology
Developmental Cell
Immunity
Med
Molecular Cell
Neuron
Structure
American Journal of Human Genetics (partner)
Biophysical Journal (partner)
Biophysical Reports (partner)
Human Genetics and Genomics Advances (partner)
Molecular Plant (partner)
Molecular Therapy (partner)
Molecular Therapy Methods & Clinical Development (partner)
Molecular Therapy Nucleic Acids (partner)
Molecular Therapy Oncology (partner)
Plant Communications (partner)
Stem Cell Reports (partner)
Trends in Biochemical Sciences
Trends in Cancer
Trends in Cell Biology
Trends in Ecology & Evolution
Trends in Endocrinology & Metabolism
Trends in Genetics
Trends in Immunology
Trends in Microbiology
Trends in Molecular Medicine
Trends in Neurosciences
Trends in Parasitology
Trends in Pharmacological Sciences
Trends in Plant Science
Physical sciences & engineering journals
Cell Biomaterials
Cell Reports Physical Science
Chem
Chem Catalysis
Chem Circularity
Device
Joule
Matter
Newton
Trends in Chemistry
Multidisciplinary journals
Cell Reports Methods
Cell Reports Sustainability
Heliyon
iScience
One Earth
Patterns
STAR Protocols
Nexus (partner)
The Innovation (partner)
Trends in Biotechnology
Trends in Cognitive Sciences
Authors
Submit article
Multi-Journal Submission
STAR Methods
Sneak Peek – Preprints
 
Reviewers
Information for reviewers
 
News & events
Newsroom
Events
Cell Symposia
Consortia Hub
Webinars
 
Multimedia
Cell Press Podcast
About
About Cell Press
Open access
COVID Hub
Sustainability hub
Inclusion and diversity
 
Careers
Cell Press Careers
Access
Subscribe
Claim
Read-It-Now
Recommend to Librarian
Publication Alerts
Collections
Best of Cell Press
Cell Press Reviews
Information
For Librarians
my account:
My Account
Privacy Center
Log out
The content on this site is intended for healthcare professionals and researchers across all fields of science.

We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the Cookie Settings for this site.
All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors.
All rights are reserved, including those for text and data mining, AI training, and similar technologies.
For all open access content, the relevant licensing terms apply.

Privacy Policy   Terms & Conditions   Accessibility   Help & Support   Contact
RELX

View PDF